app/models/translation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Translation < ActiveRecord::Base
+  belongs_to :resource
+  belongs_to :language
+  has_many :custom_pages
+
+  def add_new_version
+    Translation.create(resource: resource, language: language, version: version + 1)
+  end
+
+  def self.latest_translation(resource_id, language_id)
```

Comment:
I wouldn't make a scope for the purpose of looking up 1 record, a class level method make sense.
However, if you ever are doing `where().first`, that can be replaced by a `find_by`, so this query could be re-written: `order(version: :desc).find_by(resource_id: resource_id, language_id: language_id)`